main/src/org/destinationsol/game/planet/PlanetTiles.java

```diff
@@ -34,7 +34,7 @@
     private final Map<SurfaceDirection, Map<SurfaceDirection, List<Tile>>> myGroundTiles;
 
     public PlanetTiles(TextureManager textureManager, String groundFolder, FileHandle configFile) {
-        myGroundTiles = new HashMap<SurfaceDirection, Map<SurfaceDirection, List<Tile>>>();
+        myGroundTiles = new HashMap<>();
```

Comment:
In fact, in most places, I'm shifting such things to the class itself, so that the variable declaration becomes
`private  final Map<SurfaceDirection, Map<SurfaceDirection, List<Tile>>> myGroundTiles = new HashMap<>();`
Makes a lot more sense over there.